### DISCUSSION TITLE
Add Default Value For  TextInput In Custom Page

### DISCUSSION THREAD

**Discussion by jrpikong** [filamentphp/filament#9274] [Help]
Hi, I have new page wit type custom page, 
I want to add new TextInput with default value.

`class AddBudgetPlanner extends Page implements HasForms
{
    use InteractsWithForms, InteractsWithActions, HasPageSidebar;

    protected static string $resource = BranchResource::class;
    protected static string $view = 'filament.resources.branch-resource.pages.create-budget-planner';
    public Branch $record;
    protected static ?string $model = Branch::class;

    protected static ?string $navigationGroup = 'Budget planner';
    protected static ?string $navigationLabel = 'Add Budget Planner';

    public function getBreadcrumb(): ?string
    {
        return __('Manage Branch Budget Planner');
    }

    public function form(Form $form): Form
    {
        return $form->schema([
            \Filament\Forms\Components\TextInput::make('code')->default('some value'),
        ]);
    }`
    
    
    But TextInput not show my defailt value, I have done trype with another option implement with `afterStateHydrated` but still not work .

**Comment by leandrocfe** [comment]
Add a mount method
```php
public function mount(): void

{

$this->form->fill();

}
```

You should add a statePath as well
https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component/#adding-the-form
